main.py

```diff
@@ -794,6 +794,7 @@ def ui_access_wrapper(self, *args, **kwargs):
         admin.NumberOfDeletionRequestsHandler),
     get_redirect_route(r'/frontend_errors', FrontendErrorHandler),
     get_redirect_route(r'/logout', base.LogoutPage),
+    get_redirect_route(r'/session_begin', base.SessionBeginHandler),
```

Comment:
is it a publicly accessible link? and is it okay to make a call on this link through oppia.org after this PR is merged?

Reply:
Yes it's publicly accessible, but it's only intended to be a ping and won't do anything without an authorization header. Is that a problem?